src/app/components/RouterTabs/index.tsx

```diff
@@ -21,9 +22,11 @@ export const RouterTabs: FC<RouterTabsProps> = ({ tabs }) => {
   return (
     <>
       <Tabs value={currentTab?.to}>
-        {tabs.map(tab => (
-          <Tab key={tab.to} component={RouterLink} value={tab.to} label={tab.label} to={tab.to} />
-        ))}
+        {tabs
+          .filter(tab => tab === currentTab || tab.visible !== false)
```

Comment:
I don't like this direction. Why hide the tabs? @donouwens

Reply:
My idea was, why offer an option which is not really there, or at least irrelevant. The less visual clutter, the better.

Reply:
So far in most places we've been disabling options if unavailable (+explain in tooltip). I guess that let's user see what features we have, and if they do something that unlocks an option, they'll know where to find it.

Reply:
I'm leaning towards hiding it, but I'm find both ways. We can discuss later. 

In any case, I have now added a config flag (in `AccountDetaisPage` code) and disabled this feature.
So, no change of behavior occurs.

_However_ I think we could still merge this, since it improves the code in various small ways.

Reply:
Sure thing, that's how it behaves now.